src/Utilities/api.js

```diff
@@ -0,0 +1,95 @@
+class Geocoder {
```

Comment:
this can likely go in as a service.

Reply:
In /src/api/api.js?

Reply:
changed locations to services/locationiq/api.js